src/player.js

```diff
@@ -1,23 +1,32 @@
 import VideoPlayer from './video-player';
 import { defaultProfiles } from './config/profiles';
 import { isRawUrl } from './plugins/cloudinary/common';
+import { unsigned_url_prefix } from '@cloudinary/url-gen/backwards/utils/unsigned_url_prefix';
 
-export const getProfile = async (cloudName, profile) => {
+export const getProfile = async (profile, initOptions) => {
   if (Object.keys(defaultProfiles).includes(profile)) {
     return defaultProfiles[profile];
   }
 
-  if (isRawUrl(profile)) {
-    return await fetch(profile, { method: 'GET' }).then(res => res.json());
-  }
+  const urlPrefix = unsigned_url_prefix(
+    null,
+    initOptions.cloudName ?? initOptions.cloud_name,
+    initOptions.private_cdn,
+    initOptions.cdn_subdomain,
+    initOptions.secure_cdn_subdomain,
+    initOptions.cname,
+    initOptions.secure,
+    initOptions.secure_distribution,
+  );
```

Comment:
@jakub-roch instead of creating that list you should use `CLOUDINARY_CONFIG_PARAM`, [like in getConfig()](https://github.com/cloudinary/cloudinary-video-player/blob/edge/src/index.js#L8)

Reply:
<img width="328" alt="image" src="https://github.com/user-attachments/assets/04eecfcd-d527-4429-a5a0-3f2df4b48480">
this helper requires specific properties, some of them are ignored like ```'shorten',
  'sign_url',
  'api_secret',
  'url_suffix',
  'use_root_path',
  'auth_token'```

as I can see `cloudinaryConfig` provides same values as from initOptions